deepmd/main.py

```diff
@@ -737,6 +737,18 @@ def main_parser() -> argparse.ArgumentParser:
         default=None,
         help="Model branch chosen for changing bias if multi-task model.",
     )
+    parser_change_bias.add_argument(
+        "--skip-elementcheck",
+        action="store_true",
+        help="Enable this option to skip element checks if any error occurs while retrieving statistical data.",
+    )
+    parser_change_bias.add_argument(
+        "-mf",
+        "--min-frames",
```

Comment:
I don't think `--min-frames` shows what this option means, and it's hard to distinguished with `--numb-batch`.